sample-project/package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Using a prebuilt sample project to run tests against worked really well, I think that is a good pattern to continue since it also provides a nice demo for people to use.